cinema/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework.parsers import JSONParser
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@csrf_exempt
+def movie_list(request: HttpResponse) -> JsonResponse:
```

Comment:
In Django, requests are actually of `HttpRequest` type. And since we're using rest framework here, you can import and use `Request` and `Response` here, which would be the correct type annotation.

Reply:
Thanks!